### ISSUE TITLE
Stitches: zero value in hsl can not add style

### ISSUE THREAD

**Issue by taneba** [ben-rogerson/twin.macro#558]
First of all, thank you for the awesome library.

I found a strange bug with stitches.

If you have color settings of tailwind.config.js like
```
      colors: {
        electric: '#db00ff',
        ribbon: '#0047ff',
        cust: {
          cust1: 'hsl(0 0% 0% / 0.071)',
          cust2: 'hsl(194 99.6% 47.7% / 0.165)',
        },
      },
```

and in IndexPage (i'm using next-** boilerplate in twin.example):

```
const IndexPage = () => (
  <div>
    <h1 tw="text-ribbon">HI</h1>
    <h1 tw="text-cust-cust1">HI</h1>
    <h1 tw="text-cust-cust2">HI</h1>
  </div>
)
```

Here, text-cust-cust1 and text-cust-cust2 work with emotion.
But with Stitches, text-cust-cust2 doesn't work.

emotion:
<img width="890" alt="Screen Shot 2021-10-17 at 12 38 26" src="https://user-images.githubusercontent.com/20878498/137609972-c804f091-112d-45d6-b175-4750329bfff4.png">

stitches:
<img width="890" alt="Screen Shot 2021-10-17 at 12 37 06" src="https://user-images.githubusercontent.com/20878498/137609939-4ca6be49-ee86-46a6-bdc6-de60e75db98a.png">

So basically it seems that with stitches it couldn't generate proper css value when hsl has 0-ish value.

repro: https://github.com/taneba/twin.examples/tree/repro-stitches-hsl

cc: @peduarte

**Comment by ben-rogerson** [comment]
Closing due to lack of activity
